backend/src/main/java/cz/muni/fi/pa165/pneuservis/backend/entity/Customer.java

```diff
@@ -1,7 +1,167 @@
 package cz.muni.fi.pa165.pneuservis.backend.entity;
 
-import javax.persistence.Entity;
+import cz.muni.fi.pa165.pneuservis.backend.enums.VehicleTypeEnum;
 
-//@Entity
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Set;
+
+/**
+ * @author Zuzana Žufanová, zufinka@mail.muni.cz
+ */
+
+@Entity
 public class Customer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private String email;
+
+    @NotNull
+    private String name;
+
+    @NotNull
+    private String surname;
+
+    @NotNull
+    private String city;
+
+    @NotNull
+    private String street;
+
+    @NotNull
+    @Size (min = 5, max = 5)
+    private Integer psc;
+
+    @NotNull
+    @Size (min = 9, max = 9)
+    private Integer phoneNumber;

```

Comment:
There are phone numbers with different lengths than 9, and some of them start with a "0" which would be a pain to implement with an Integer. It should be a String (so that it supports "+" as well) with a check performed on some other layer.